.rubocop.yml

```diff
@@ -45,6 +48,7 @@ Metrics/AbcSize:
 
 Metrics/BlockLength:
   Exclude:
+    - "lib/bootstrap_form/inputs/base.rb"
```

Comment:
I guess we can leave this one in here, since it applies to the whole file anyway. But I should look at the rule and where it came from. I'm a fan of RuboCop, but not of these rules that put arbitrary limits on the size of things.